src/jsx/import/content/login.jsx

```diff
@@ -11,152 +11,152 @@
+                            // boolean to hide ajax spinner
+                            this.setState({ display_spinner: false });
 
                             break;
+                        }
                     }
                 }
-            }
-
-        }.bind(this),
-      // asynchronous callback: ajax 'fail' promise
-        function (asynchStatus, asynchError) {
-            if (asynchStatus) {
-                this.setState({ajax_fail_status: asynchStatus});
-                console.log('Error Status: ' + asynchStatus);
-            }
-
-            if (asynchError) {
-                this.setState({ajax_fail_error: asynchError});
-                console.log('Error Thrown: ' + asynchError);
-            }
-
-          // boolean to hide ajax spinner
-            this.setState({display_spinner: false});
-        }.bind(this),
-      // pass ajax arguments
-        ajaxArguments);
-    },
-    updateUsername: function(event) {
-        this.setState({value_username: event.target.value});
-    },
-    componentWillMount: function() {
-      // update redux store
-        this.props.dispatchLayout(setLayout({'layout': 'login'}));
-    },
-    componentDidMount: function() {
-        this.props.dispatchSpinner(setSpinner({'spinner': false}));
-    },
-  // triggered when 'state properties' change
-    render: function() {
-      // local variables
+            },
+            // asynchronous callback: ajax 'fail' promise
+            (asynchStatus, asynchError) => {
+                if (asynchStatus) {
+                    this.setState({ ajax_fail_status: asynchStatus });
+                    console.log(`Error Status: ${asynchStatus}`);
+                }
+
+                if (asynchError) {
+                    this.setState({ ajax_fail_error: asynchError });
+                    console.log(`Error Thrown: ${asynchError}`);
+                }
+
+                // boolean to hide ajax spinner
+                this.setState({ display_spinner: false });
+            },
+            // pass ajax arguments
+            ajaxArguments,
+        );
+    }
+    updateUsername(event) {
+        this.setState({ value_username: event.target.value });
+    }
+    componentWillMount() {
+        // update redux store
+        this.props.dispatchLayout(setLayout({ layout: 'login' }));
+    }
+    componentDidMount() {
+        this.props.dispatchSpinner(setSpinner({ spinner: false }));
+    }
+    // triggered when 'state properties' change
+    render() {
+        // local variables
         var redirect = null;
-        var AjaxSpinner = this.getSpinner();
+        const AjaxSpinner = this.getSpinner();
 
-      // backend validation
+        // backend validation
         if (!this.state.validated_login_server) {
-            var loginNote = <div className='invalid-pop'>
+            var loginNote = (
+                <div className='invalid-pop'>
                 Invalid user, or password.
-            </div>;
+                </div>);
```

Comment:
Does it make more sense to have:

```javascript
            var loginNote = (
                <div className='invalid-pop'>
                 Invalid user, or password.
                </div>
            );
```

Reply:
Sure it does!
I think it even looks more beautiful, to be honest